extensions/sql-database-projects/src/models/tree/projectTreeItem.ts

```diff
@@ -26,14 +26,17 @@ export class ProjectRootTreeItem extends BaseProjectTreeItem {
 	fileChildren: { [childName: string]: (fileTree.FolderNode | fileTree.FileNode) } = {};
 	project: Project;
 	fileSystemUri: vscode.Uri;
+	projectNodeName: string;
 
 	constructor(project: Project) {
-		super(vscode.Uri.parse(path.basename(project.projectFilePath, sqlprojExtension)), undefined);
+		super(vscode.Uri.parse(path.basename(project.projectFilePath, sqlprojExtension)), vscode.Uri.file(project.projectFilePath), undefined);
 
 		this.project = project;
 		this.fileSystemUri = vscode.Uri.file(project.projectFilePath);
-		this.databaseReferencesNode = new DatabaseReferencesTreeItem(this);
-		this.sqlCmdVariablesNode = new SqlCmdVariablesTreeItem(this);
+		this.projectNodeName = path.basename(project.projectFilePath, sqlprojExtension);
```

Comment:
Noting for later: this'll probably need to get read from the project file rather than just assuming that the .sqlproj filename is the project name